### PR TITLE
238 refactor organize tests

### PR DESCRIPTION
**Pull request by digicosmos86**
Mainly just cleaned up the tests a bit. Using one file for dataset simulation across the board.

### CODE REVIEW

File: tests/test_distribution_utils.py [hunk 1]
```diff
@@ -68,3 +69,86 @@ def test_lapse_distribution():
     random_sample_b = rv.rng_fn(rng2, *[0.5, 0.5, 0.5, 0.3], 0.05, 10)
 
     np.testing.assert_array_equal(random_sample_a, random_sample_b)
+
+
+def test_apply_param_bouds_to_loglik():
```
Comment (AlexanderFengler) [inline tests/test_distribution_utils.py:74]: test_apply_param_bouds_to_loglik --> test_apply_param_bounds_to_loglik ?

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import numpy as np
+import pandas as pd
+import ssms
+
+import hssm
+
+
+@pytest.fixture(scope="module")
+def data_ddm():
+    v_true, a_true, z_true, t_true = [0.5, 1.5, 0.5, 0.5]
```
Comment (AlexanderFengler) [inline tests/conftest.py:12]: Just a note: Might be good to test a range of parameter sets here for robustness in some cases. Those parameters are hard-coded here, and we might miss some strange things with the likelihoods if we fixate only on this (and the one below) parameter set.

**Review by AlexanderFengler — APPROVED** [review]
This looks fine to me.

File: tests/test_distribution_utils.py (hunk 1 above)
Comment (digicosmos86) [reply tests/test_distribution_utils.py:74]: Nice catch!

Comment (digicosmos86) [reply tests/test_distribution_utils.py:74]:
```suggestion
def test_apply_param_bounds_to_loglik():
```

**Pull request merged**